### ISSUE TITLE
Can the commands be renamed to indicate what they are doing?

### ISSUE THREAD

**Issue by ghost** [leehooi/vs-external-tools#5]

**Comment by mbaas2** [comment]
+1 (I would find that useful, too)

**Comment by FatehAK** [comment]
I second that feature

**Comment by dreftymac** [comment]
Yes. That would be very useful. Adding a new setting such as `vs-external-tools.externalCommand1.caption` with a human-readable label that allows the user to distinguish the tool would greatly enhance the usability of this VSCode extension.
